l10n_it_fiscalcode/wizard/compute_fc.py

```diff
@@ -185,16 +178,23 @@ def compute_fc(self):
                 or not f.sex
             ):
                 raise UserError(_("One or more fields are missing"))
+
             nat_code = self._get_national_code(
                 f.birth_city.name, f.birth_province.code, f.birth_date
             )
             if not nat_code:
                 raise UserError(_("National code is missing"))
-            c_f = build(
+
+            if isinstance(f.birth_date, datetime.date):
```

Comment:
Il campo `birth_date` può essere una stringa?

Reply:
La libreria proposta deve ricevere l valore di birth_date come stringa

Reply:
Ma in che caso il campo `birth_date = fields.Date("Date of birth", required=True)` può contenere una stringa?

Reply:
Scusami ma credo che non ci stiamo capendo; la libreria proposta vuole il valore del campo `birth_date` come <ins>stringa</ins>, [metodo calcolo CF](https://github.com/fabiocaccamo/python-codicefiscale/blob/main/codicefiscale/codicefiscale.py#L365) richiede che il valore di `birth_date` sia `datetime` o `str` qui gli arriva come data e lo converto in str

Reply:
Già 😅
Il campo "f.birth_date" come potrebbe contenere una stringa se è definito come      fields.Date()?
A che serve "if isinstance(f.birth_date, datetime.date):"?
Trasformalo in stringa direttamente.

Reply:
Può contenere una stringa perché nei test vien passato come tale, mente da ui viene caricato come fields.Date(). Quindi invece che modificare i test ho preferito fare un controllo e una conversione. I test, per quanto possibile non andrebbero modificati, perché se sono corretti sono la garanzia che il nuovo codice non introduce regressioni

Reply:
@sergiocorato aggiornato i test e rimosso il check nel wizard

Reply:
Ottimo, grazie